node/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+name = "node"
+version = "0.1.0"
+edition = "2021"
+autobins = false
+description = "An implementation of dusk-blockchain node in pure Rust"
+categories = ["cryptography", "cryptography::cryptocurrencies"]
+keywords = ["dusk", "cryptocurrency", "blockchain", "node"]
+license = "MPL-2.0"
+
+[[bin]]
+name = "rusk-node"
+path = "src/bin/main.rs"
+
+
+[dependencies]
+tracing = "0.1"
+hex = { version= "0.4.3"}
+dusk-consensus = "0.1.0"
```

Avoid to stick to a patch version, change this to be "0.4" and go for a short dependency declaration 
```suggestion
hex = "0.4"
dusk-consensus = "0.1"
```